generator/plugins/proto2gql/plugin.go

```diff
@@ -39,12 +39,35 @@ func (p *Plugin) Init(config *generator.GenerateConfig, plugins []generator.Plug
 	}
 	p.generateConfig = config
 	p.config = cfg
+
+	err = p.parseImports()
+	if err != nil {
+		return errors.Wrap(err, "failed to decode imports")
+	}
+
 	err = p.normalizeGenerateConfigPaths()
 	if err != nil {
 		return errors.Wrap(err, "failed to normalize config paths")
 	}
+
+	return nil
+}
+
+func (p *Plugin) parseImports() error {
+	for _, pluginsConfigsImports := range p.generateConfig.PluginsConfigsImports {
+		cfg := new([]*ProtoFileConfig)
+		err := mapstructure.Decode(pluginsConfigsImports["proto2gql_files"], cfg)
```

Comment:
can you make a constant for "proto2gql_files" string?

Reply:
done